### DISCUSSION TITLE
在openclash里点了应用之后，有时候路由器的CPU占用会特别高，造成路由器死机甚至重启。

### DISCUSSION THREAD

**Discussion by fransewoo** [vernesong/OpenClash#2758] [Q&A]
在全局设置或策略组管理设置里，点击应用配置之后，会有很大概率出现。我有一次晚上还好好的，第二天早上就出现CPU占用过高。SSH登陆路由器，top 命令里查看，只要进程里出现：
14317     1 nobody   S     792m 691%   0% /etc/openclash/clash -d /etc/openclash -f /etc/openclash/keke.yaml
CPU会被占满，造成路由器假死，等一段时间路由器会死机重启，重启一会马上出现同样状况。
只好在重启完上线那一会赶紧把openclash关闭，路由器的cpu占用就会很低，一般5%左右。
PS：keke.yaml是我的配置文件订阅。
固件版本 ：OpenWrt 10.18.2022 by Kiddin' / LuCI Master git-1210696
OpenClash版本：Current Version v0.45.59-beta
路由器是华硕RT-AC58U（RT-ACRH13），具体配置如下：
Qualcomm Atheros IPQ4018 | CPU 717 MHz | Flash 128 MB | RAM 128 MB | WLAN Hardware - Qualcomm Atheros IPQ4018 。

**Comment by lozzo** [comment]
现在也有这个问题

**Comment by vernesong** [comment]
把日志开着，看是不是回环了
